kong/clustering/rpc/manager.lua

```diff
@@ -141,6 +142,120 @@ function _M:_find_node_and_check_capability(node_id, cap)
 end
 
 
+-- CP => DP
+function _M:_handle_meta_call(c, node_id)
+  local data, typ, err = c:recv_frame()
```

Comment:
I think at least we can reuse some functions from `socket.lua`, instead of manipulating the websocket directly

Reply:
No, it is a sync action, different with async action in socket.lua, I think that there is little code to reuse.